runtime/hydradx/src/assets.rs

```diff
@@ -985,3 +992,141 @@ impl pallet_xyk::Config for Runtime {
 	type NonDustableWhitelistHandler = Duster;
 	type OracleSource = XYKOracleSourceIdentifier;
 }
+
+parameter_types! {
+	pub const ReferralsPalletId: PalletId = PalletId(*b"referral");
+	pub RegistrationFee: (AssetId,Balance, AccountId)= (NativeAssetId::get(), 222_000_000_000_000, TreasuryAccount::get());
+	pub const MaxCodeLength: u32 = 7;
+	pub const ReferralsOraclePeriod: OraclePeriod = OraclePeriod::Short;
+	pub const ReferralsSeedAmount: Balance = 100_000_000_000_000;
+}
+
+impl pallet_referrals::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type AuthorityOrigin = EnsureRoot<AccountId>;
+	type AssetId = AssetId;
+	type Currency = FungibleCurrencies<Runtime>;
+	type Convert = ConvertViaOmnipool;
+	#[cfg(not(feature = "runtime-benchmarks"))]
+	type PriceProvider =
+		OraclePriceProviderUsingRoute<Router, OraclePriceProvider<AssetId, EmaOracle, LRNA>, ReferralsOraclePeriod>;
+	#[cfg(feature = "runtime-benchmarks")]
+	type PriceProvider = ReferralsDummyPriceProvider;
+	type RewardAsset = NativeAssetId;
+	type PalletId = ReferralsPalletId;
```

Comment:
any reason why we dont use default pallet id?

Reply:
we always did this for other pallets, just did the same.